ecommerce/extensions/payment/processors/payu.py

```diff
@@ -117,12 +127,26 @@ def get_description(basket):
             return None
 
         descriptions = []
-        separator = ' | '
         for line in basket.lines.all():
             if line.product.get_product_class() == seat_class:
-                descriptions.append(line.product.course_id)
+                splitted_course_id = line.product.course_id.split('+', self.MAX_SPLITS)
+                descriptions.append(splitted_course_id[1])
 
-        return separator.join(descriptions)
+        return self.DESCRIPTION_SEPARATOR.join(descriptions)
+
+    @staticmethod
+    def get_dni(request, basket):
+        """
+        Returns the buyer user (edxapp user) dni
+        Returns None if an exception occurs
+        """
+        try:
+            buyer_user = User.objects.get(email=basket.owner.email)
+            response = buyer_user.account_details(request)
+            dni = next(field for field in response['extended_profile'] if field["field_name"] == "dni")
+            return dni['field_value']
+        except (ConnectionError, SlumberBaseException, Timeout, StopIteration, KeyError, ObjectDoesNotExist):
```

Comment:
Maybe log something here to have an evidence in the case get_dni fails (it's something critical for Zulma I assume)

Reply:
Logger added @jfavellar90